lua/resession/init.lua

```diff
@@ -105,10 +107,34 @@ M.list = function(opts)
     end
     entries = uv.fs_readdir(fd)
   end
+
   uv.fs_closedir(fd)
+
+  -- Apply order
+  if config.load_order == "filename" then
+    -- Sort by filename
+    table.sort(ret)
+  elseif config.load_order == "modification_time" then
+    -- Sort by modification_time
+    table.sort(ret, function(a, b)
+      local file_a = uv.fs_stat(session_dir .. "/" .. a .. ".json")
+      local file_b = uv.fs_stat(session_dir .. "/" .. b .. ".json")
+      return file_a.mtime.sec > file_b.mtime.sec
+    end)
+  elseif config.load_order == "creation_time" then
+    -- Sort by creation_time in descending order (most recent first)
+    table.sort(ret, function(a, b)
+      local file_a = uv.fs_stat(session_dir .. "/" .. a .. ".json")
+      local file_b = uv.fs_stat(session_dir .. "/" .. b .. ".json")
+      return file_a.ctime.sec > file_b.ctime.sec
```

Comment:
`ctime` is actually the "change time", or time that the file was last changed (e.g. permissions or other metadata). You probably want to use `birthtime`.